packages/appcues-custom-previewer/android/build.gradle

```diff
@@ -0,0 +1,97 @@
+buildscript {
+  // Buildscript is evaluated before everything else so we can't use getExtOrDefault
+  def kotlin_version = rootProject.ext.has("kotlinVersion") ? rootProject.ext.get("kotlinVersion") : project.properties["AppcuesCustomPreviewer_kotlinVersion"]
+
+  repositories {
+    google()
+    mavenCentral()
+  }
+
+  dependencies {
+    classpath "com.android.tools.build:gradle:7.2.1"
```

Comment:
must work but I think we've seen issues lower than 7.4.2, may want to just bump it up to match whatever the RN module is using.

Reply:
Updated to 7.4.2